spec/models/cart_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'rails_helper'
+
+describe Cart do
+
+  before do
+    item1 = create(:item, id: 1, price: 10)
+    item2 = create(:item, id: 2, price: 10)
+    item3 = create(:item, id: 3, price: 10)
+    @cart = Cart.new("1" => 2, "2" => 3, "3" => 4)
+  end
+
+  describe '#contents' do
+    it 'stores contents in a hash' do
+      expect(@cart.contents).to eq({"1" => 2, "2" => 3, "3" => 4})
```

Comment:
Wondering if this is a robust test--in line 9, you created the cart as a hash, and then in line 14 are checking to see if it is a hash--might be more organic if it was capybara'd out.